### DISCUSSION TITLE
what is this github thing called discussions and where do i get one?

### DISCUSSION THREAD

**Discussion by cornernote** [johnlindquist/kit#1102] [General]
This discussions section is kick ass!  

Where did you get it, and how can I get mine?! :)

**Comment by johnlindquist** [comment]
@cornernote here you go: https://docs.github.com/en/discussions/quickstart#enabling-github-discussions-on-your-repository

The cool part is it had a graphql api. That's what drives the "share" category of discussions into https://scriptkit.com/scripts
